lib/topics_filter.rb

```diff
@@ -388,6 +419,138 @@ def filter_by_number_of_views(min: nil, max: nil)
     filter_by_topic_range(column_name: "topics.views", min:, max:)
   end
 
+  # users:a,b => any of a or b participated in the topic
+  # users:a+b => both a and b participated in the topic
+  def filter_users(values:)
+    values.each do |value|
+      if value.include?("+")
+        usernames = value.split("+")
+        require_all = true
+        if value.include?(",")
+          # no mix and match
+          @scope = @scope.none
+          next
+        end
+      else
+        usernames = value.split(",")
+        require_all = false
+        if value.include?("+")
+          # no mix and match
+          @scope = @scope.none
+          next
+        end
+      end
+
+      usernames = usernames.map(&:downcase).reject(&:blank?)
+
+      if usernames.empty?
+        @scope = @scope.none
+        next
+      end
+
+      user_ids = User.where(staged: false).where("username_lower IN (?)", usernames).pluck(:id)
```

Comment:
```suggestion
      user_ids = User.not_staged.where("username_lower IN (?)", usernames).pluck(:id)
```

Also should we restrict to `human_users` here?Not sure if it will be useful to filter for bot users, but maybe